### PR TITLE
Question5: create both button "back" and "submit".

### PR DESCRIPTION
**Pull request by bhagya099**

### CODE REVIEW

File: pages/form/Question5.js [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
+  SubmitButton,
+} from '../../components/LinkButton/LinkButton';
+import capitalize from '../../utils/capitalize';
+import Q5Progress from '../../public/images/progress-bar/q5-progress-bar.svg';
+import { sendFormResponse } from '../../utils/dbApi';
 
 export default function Question5() {
   const { answers, setAnswers } = useForm();
   const [department, setDepartment] = useState(answers.department);
+  const [incentive, setIncentive] = useState(answers.incentive);
```
Comment (hqtan) [inline pages/form/Question5.js:19]: I don't think we capture anything with `incentive` variable in Question 5; the response for Question 5 should be captured in `department` variable, right?
So I think you can remove this line.
```suggestion
```

File: pages/form/Question5.js [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
 
+  const saveAnswers = () => setAnswers((prev) => ({ ...prev, incentive }));
```
Comment (hqtan) [inline pages/form/Question5.js:21]: `saveAnswers` should be saving the response in `department` variable, not `incentive`; you could inadvertently be overwriting users' response in Question 4 !
```suggestion
  const saveAnswers = () => setAnswers((prev) => ({ ...prev, department }));
```

**Review by hqtan — CHANGES_REQUESTED** [review]
Hi @bhagya099 !
Many thanks for your PR! :bow: 
I tested your changes on the preview site, and noticed that the form response for Question 5  isn't being saved when the 'Back' button was clicked; I have given some suggestions to remedy the problem. Please review and fix.

Thank you!

**Comment by hqtan** [comment]
@bhagya099 ,

Another thing I noted is the use of single quotes (') for quotations in the changes.
The general convention in our codebase is to use double quotes (") - you can confirm that by viewing the quotations in other form pages.
Please update the PR to use double quotes for quotations.

Thanks! :bow:

**Comment by hqtan** [comment]
[DEV: [SML] update page 5](https://trello.com/c/lj5HHd1L/193-dev-sml-update-page-5)

**Comment by bhagya099** [comment]
@hqtan,

Thank you for your feedback. I will do changes.

**Review by hqtan — APPROVED** [review]

**Review by hqtan — APPROVED** [review]
:+1:

**Pull request merged**